packages/firebase_database/firebase_database_web/lib/src/query_web.dart

```diff
@@ -64,8 +66,6 @@ class QueryWeb extends QueryPlatform {
     return instance;
   }
 
-  final Map<String, int> _streamHashCodeMap = {};
```

Comment:
It was in the local context of the Query so a new one was creating for each instance 🦢